spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -32,20 +32,26 @@
 @ConfigurationProperties(prefix = "spring.cloud.stream.kinesis.binder")
 public class KinesisBinderConfigurationProperties {
 
-	private String[] headers = new String[] { };
+	private String[] headers = new String[] {};
 
+	@Deprecated
```

Comment:
Any reason why do we deprecate this option?
I see you still use it in the `KinesisStreamProvisioner`

Reply:
Yes, I'll reuse the old properties. I've already fixed it and will push it as soon as I get some more time.